src/opensuse/15.4/helix/amd64/Dockerfile

```diff
@@ -26,6 +26,7 @@ RUN zypper ref && \
         lldb-devel \
         llvm \
         lttng-ust-devel \
+        pigz \
```

Comment:
This is a helix image, so we don't need this here.

Reply:
```suggestion
```